rootex/core/renderer/shaders/god_rays_pixel_shader.hlsl

```diff
@@ -0,0 +1,63 @@
+#define THRESHOLD 0.95f
+
+cbuffer cbGodRays : register(b0)
```

Comment:
oh noooo don't bind at `b0`, we might overwrite some other CB, use the constant defined in `register_*.h`